Source/Views/TextViews/SPTextView.m

```diff
@@ -74,7 +74,7 @@
 #define SP_CQ_COPY_AS_RTF_MENU_ITEM_TAG          1001
 #define SP_CQ_SELECT_CURRENT_QUERY_MENU_ITEM_TAG 1002
 
-#define SP_SYNTAX_HILITE_BIAS 2000
+#define SP_SYNTAX_HILITE_BIAS 1000
```

Comment:
Made this smaller as we only need space around current text now, because everything else is processed when inserted/pasted